pygorithm/data_structures/tree.py

```diff
@@ -405,6 +405,17 @@ def number_of_nodes(self, root):
 
         return left_number + right_number + 1
     
+    def getLeafCount(node): 
```

Comment:
Change function name to `get_leaf_count` so as follow Python PEP8 styling

Reply:
Done